source/fuzz/call_graph.h

```diff
@@ -43,6 +43,18 @@ class CallGraph {
   // invokes.
   std::set<uint32_t> GetIndirectCallees(uint32_t function_id) const;
 
+  // Returns the ids of all the functions in the graph in a topological order,
+  // in relation to the function calls, which are assumed to be recursion-free.
+  const std::vector<uint32_t>& GetFunctionsInTopologicalOrder() const {
+    return functions_in_topological_order_;
+  }
+
+  // Returns the maximum loop nesting depth from which |function_id| can be
+  // called from.
```

Comment:
"from which |function_id| can be called from." -> "from which |function_id| can be called."

Also be clear on the fact that this is inter-procedural, and be clear that it's a static analysis, i.e. the reported nesting depth might not actually be possible at runtime.

Reply:
Can you please augment the top-level comment for this class with two things:

(1) The fact that the module is assumed to be recursion-free
(2) The fact that the call graph is immutable, and thus needs to be computed afresh if the module changes

Reply:
Done